### PR TITLE
fix(Disney+): add new domain

### PR DESCRIPTION
**Pull request by sarcasterXO**
## Description 
Added `www.hotstar.com` domain. As in India, whenever you try to open the website using `www.disneyplus.com` it redirects to `www.hotstar.com/in` and the presence stops by saying `Browsing...` even if I start watching a video. 

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

### CODE REVIEW

File: websites/D/Disney+/metadata.json [hunk 1]
```diff
@@ -19,7 +19,10 @@
 		"da_DK": "Disney+ er det eksklusive hjem for dine yndlingsfilm og -tv-serier fra Disney, Pixar, Marvel, Star Wars, National Geographic og Star. Begynd at se i dag.",
 		"pt_BR": "O Disney+ é o serviço exclusivo para ver os seus filmes e programas preferidos da Disney, Pixar, Marvel, Star Wars e National Geographic. Comece a assistir hoje mesmo."
 	},
-	"url": "www.disneyplus.com",
+	"url": [
+		"www.disneyplus.com",
+		"www.hotstar.com"
+	],
 	"regExp": "([a-z0-9-]+[.])*disneyplus[.]com[/]|([a-z0-9-]+[.])*hotstar[.]com[/]",
 	"version": "1.4.19",
```
Comment (theusaf) [inline websites/D/Disney+/metadata.json:27]:
```suggestion
	"version": "1.4.20"
```

**Review by theusaf — CHANGES_REQUESTED** [review]

**Comment by sarcasterXO** [comment]
> That means the url is already supported, and just an FYI, the `url` field is ignored when the `regExp` field is present. So this PR won't fix the issue.

So, you mean I should remove the `regExp` field and only use `url`?

File: websites/D/Disney+/metadata.json (hunk 1 above)
Comment (EncryptedDev) [reply websites/D/Disney+/metadata.json:27]: Missing trailing comma

Comment (theusaf) [reply websites/D/Disney+/metadata.json:27]: Fixed.

**Comment by sarcasterXO** [comment]
> No, as I said, the url is already supported. 
> Basically, this isn't a solution to your issue.


Okay, I'll test again and commit the new changes.

**Comment by sarcasterXO** [comment]
@Slowlife01 is correct, I tested it myself with and without the new domain and it works fine. I don't know what's wrong with the current extension though as while testing, it works fine.

**Pull request closed without merging**